src/frontend/screens/Library/components/InstallModal/SideloadDialog/index.tsx

```diff
@@ -208,31 +208,43 @@ export default function SideloadDialog({
     return backdropClick()
   }
 
-  const fileFilters = {
-    Windows: [
-      { name: 'Executables', extensions: ['exe', 'msi'] },
-      { name: 'Scripts', extensions: ['bat'] },
-      { name: 'All', extensions: ['*'] }
-    ],
-    linux: [
-      { name: 'AppImages', extensions: ['AppImage'] },
-      { name: 'Other Binaries', extensions: ['sh', 'py', 'bin'] },
-      { name: 'All', extensions: ['*'] }
-    ],
-    Mac: [
-      { name: 'Apps', extensions: ['App'] },
-      { name: 'Other Binaries', extensions: ['sh', 'py', 'bin'] },
-      { name: 'All', extensions: ['*'] }
-    ]
-  }
+  const fileFilters = useCallback((platform: InstallPlatform) => {
+    switch (platform) {
+      case 'Windows':
+      case 'windows':
+      case 'Win32':
+        return [
+          { name: 'Executables', extensions: ['exe', 'msi'] },
+          { name: 'Scripts', extensions: ['bat'] },
+          { name: 'All', extensions: ['*'] }
+        ]
+      case 'linux':
+        return [
+          { name: 'AppImages', extensions: ['AppImage'] },
+          { name: 'Other Binaries', extensions: ['sh', 'py', 'bin'] },
+          { name: 'All', extensions: ['*'] }
+        ]
+      case 'osx':
+      case 'Mac':
+        return [
+          { name: 'Apps', extensions: ['App'] },
+          { name: 'Other Binaries', extensions: ['sh', 'py', 'bin'] },
+          { name: 'All', extensions: ['*'] }
+        ]
+      // FIXME: Can these happen?
+      case 'Android':
```

Comment:
Maybe just use `default` here

Reply:
I don't like `default` here: In case we ever add a new install platform, I'd rather have this switch statement error out until that platform is explicitly added to the case list than it getting silently ignored